hack/ci/Vagrantfile

```diff
@@ -31,6 +30,16 @@ Vagrant.configure("2") do |config|
         oci-seccomp-bpf-hook \
         openssl \
         podman
+
+      echo "nameserver 8.8.8.8" > /etc/resolv.conf
+      modprobe br_netfilter
+      ip6tables --list >/dev/null
+      iptables -F
+      sysctl -w net.ipv4.conf.all.route_localnet=1
+      sysctl -w net.ipv4.ip_forward=1
+      sysctl -w net.bridge.bridge-nf-call-iptables=1
+      sysctl -w fs.inotify.max_user_watches=1048576
+      iptables -t nat -I POSTROUTING -s 127.0.0.0/8 ! -d 127.0.0.0/8 -j MASQUERADE
```

Comment:
Mind adding a comment or a description on the commit message of why all of this is needed?

Reply:
uhm... why do we need that postrouting iptables rules?